client/components/activity-card-list/index.jsx

```diff
@@ -21,14 +23,24 @@ import ActivityCard from 'calypso/components/activity-card';
 import Filterbar from 'calypso/my-sites/activity/filterbar';
 import getActivityLogFilter from 'calypso/state/selectors/get-activity-log-filter';
 import Pagination from 'calypso/components/pagination';
+import QueryActivityLogRetentionPolicy from 'calypso/components/data/query-activity-log-retention-policy';
 import QueryRewindCapabilities from 'calypso/components/data/query-rewind-capabilities';
 import QueryRewindState from 'calypso/components/data/query-rewind-state';
+import RetentionLimitUpsell from 'calypso/components/activity-card-list/retention-limit-upsell';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
+const getRetentionPolicyRequestStatus = ( state, siteId ) => {
```

Comment:
This may warrant its own file eventually, but I wasn't sure if that time would be now or later.

Reply:
if it is only used here I think YAGNI applies 👍